openslides/core/serializers.py

```diff
@@ -78,7 +78,7 @@ class ProjectorSerializer(ModelSerializer):
     """
 
     elements = JSONSerializerField(validators=[elements_validator])
-    elements_preview = JSONSerializerField(validators=[elements_array_validator])
+    elements_preview = JSONSerializerField(validators=[elements_validator])
```

Comment:
This seams wrong. The preview field is an array. Why did you change this?

Reply:
Its an array of single elements. The preview just supports one element at a time, so you have a list of elements, that will be projected.

Reply:
You can have a look in the client, then it might be clear (note, that the "append to preview" button is missing, but you can use next/previous)